kqueen_ui/blueprints/ui/views.py

```diff
@@ -577,11 +577,11 @@ class ClusterCreate(KQueenView):
     def handle(self):
         # Get all necessary objects from backend
         _provisioners = self.kqueen_request('provisioner', 'list')
-        unknown_state = app.config['PROVISIONER_UNKNOWN_STATE']
-        ok_state = app.config['PROVISIONER_OK_STATE']
         provisioners = [
-            p for p in _provisioners
-            if p.get('state', unknown_state) == ok_state
+            p
```

Comment:
Why do you need to add the line break? `p for p in _provisioners` fits the row perfectly